packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -513,27 +521,29 @@ export const ReactEditor = {
     }
 
     if (!textNode) {
```

Comment:
I don't think this is correct.  You can't just delete this exception in case extractMatch [sic] is false.  I think the code should be
```
    if (!textNode) {
      if (exactMatch) {
        return null as T extends true ? Point | null : Point
      }
      throw new Error(
        `Cannot resolve a Slate point from DOM point: ${domPoint}`
      );
    }
```
Of course you would only notice this when some other bug messes up the input to this function...

Reply:
you're right,  I forgot to restore the code while all errors that were thrown manually in `slate-react` and replaced them with deselection in my own forked slate.